tests/UI/pages/BO/design/themeAndLogo/advancedCustomization/parentChildTheme.js

```diff
@@ -0,0 +1,21 @@
+require('module-alias/register');
+const CommonPage = require('@pages/commonPage');
+
+/**
+ * Parent Child Theme dev doc page, contains functions that can be used on the page
+ * @class parentChildTheme
+ * @extends CommonPage
+ */
+class ParentChildTheme extends CommonPage {
```

Comment:
```suggestion
class ParentChildTheme extends BOBasePage {
```
Is there a raison why we extending from CommonPage ?

Reply:
I didn't extend from the `BOBasePage` because The Parent child theme is an external page: https://devdocs.prestashop.com/1.7/themes/reference/template-inheritance/parent-child-feature/

Thank you!

Reply:
Hi @boubkerbribri,

I removed this page as it is an external page & I replaced it by checking only the href.

Thank you!